lib/portage/util/__init__.py

```diff
@@ -2006,3 +2008,9 @@ def read_ld_so_conf(path):
     rval.append("/lib")
 
     return [normalize_path(x) for x in rval if x]
+
+
+def no_color(nocolor: str, no__color: Optional[str]) -> bool:
```

Comment:
Why __? Can we just use a different function name to make this less confusing? "has_color"? "allow_color"?